INSTALL.md

```diff
@@ -2,11 +2,28 @@
 ### Running using mvn (local develpment) or Running using docker (deployment)
 See the `INSTALL.md` files in the [backend](./backend/INSTALL.md) and [frontend](./frontend/INSTALL.md) folder.
 ### Running using helm (deployment)
-1. Run the application:
+1. Configure and run the application:
+> **Note:**   
+> Before the command below can be executed, the values in the curly brackets need to be adjusted to your requirements.
 ```shell
-cd charts/puris/
-
-helm install puris --namespace puris --create-namespace . --set frontend.ingress.enabled=true --set backend.ingress.enabled=true
+helm install puris charts/puris \
+    --namespace puris \
+    --create-namespace \
+    --set frontend.ingress.enabled=true \
```

Comment:
Did you try to replace the values and run it locally? For me this results in problems with the ingress:
- Ingress needs lowercase (internal validation I wasn't aware of).
- when setting values in an array, the others are reset ([bug / feature discussion](https://github.com/helm/helm/issues/5711#issuecomment-636177594)) -> we should at most set ingress to enabled and list the respective values that might be configured in a values file above the install.md